### PR TITLE
ES2015ify tests and add XO

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-var assert = require('assert');
-var fs = require('fs');
-var getUrls = require('./');
+import fs from 'fs';
+import test from 'ava';
+import fn from './';
 
-describe('should find urls in fixture file', function () {
-	var expected = [
+test('should get unique cleaned-up urls from a string', t => {
```
Comment (SamVerschueren) [inline test.js]: `get unique cleaned-up urls from a string`

File: test.js [hunk 2]
```diff
@@ -13,53 +12,47 @@ describe('should find urls in fixture file', function () {
 		'http://github.com'
 	];
 
-	it('should get unique cleaned-up urls from a string', function () {
-		assert.deepEqual(getUrls(fs.readFileSync('fixture.txt', 'utf8')), expected);
-	});
+	t.same(fn(fs.readFileSync('fixture.txt', 'utf8')), expected);
```
Comment (SamVerschueren) [inline test.js]: I would place the array directly in this line. Not sure what @sindresorhus likes though.

File: test.js [hunk 3]
```diff
@@ -13,53 +12,47 @@ describe('should find urls in fixture file', function () {
...
 });
 
-describe('stripFragment option', function () {
-	it('should not strip fragments when skipFragments is set to false', function () {
-		var text = 'You can read http://www.foobar.com/document.html#about for more info';
-		var expected = ['http://foobar.com/document.html#about'];
-		var actual = getUrls(text, {stripFragment: false});
-		assert.deepEqual(actual, expected);
-	});
+test('should not strip fragments when skipFragments is set to false', t => {
```
Comment (SamVerschueren) [inline test.js]: remove `should`

File: test.js (hunk 1 above)
Comment (kevva) [inline test.js]: Yeah, didn't care about changing any wording, just rewrite them to use `ava`.

File: test.js [hunk 4]
```diff
@@ -13,53 +12,47 @@ describe('should find urls in fixture file', function () {
...
+	const text = 'You can read http://www.foobar.com/document.html#about for more info';
+	const expected = ['http://foobar.com/document.html#about'];
+	const actual = fn(text, {stripFragment: false});
+	t.same(actual, expected);
```
Comment (SamVerschueren) [inline test.js]:
```js
t.same(fn(text, {stripFragment: false}), ['http://foobar.com/document.html#about']); 
```

looks better to me. Wdyt?

File: test.js (hunk 2 above)
Comment (kevva) [inline test.js]: Same as above, didn't do much else than rewrite them to ES2015 :).

File: test.js (hunk 4 above)
Comment (kevva) [inline test.js]: Yup, I agree.

File: test.js (hunk 2 above)
Comment (SamVerschueren) [inline test.js]: Same as all the rest of my comments :). I'll leave it to @sindresorhus to decide if those should be rewritten.

**Comment by sindresorhus** [comment]
Woot, @kevva is back! :)

I agree with Sam's comments.

**Comment by kevva** [comment]
Aight, should be resolved now!

**Comment by sindresorhus** [comment]
Awesome! :)

**Pull request merged**